### PR TITLE
add json for case 510612

### PR DESCRIPTION
**Pull request by zhouying7780**
@wsun1

### CODE REVIEW

File: pods/pod_with_special_fsGroup.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        "name": "hello-pod",
+        "labels": {
+            "name": "hello-pod"
+        }
+    },
+    "spec": {
+        "containers": [
+            {
+                "name": "hello-pod",
+                "image": "bmeng/hello-openshift",
```
Comment (wsun1) [inline pods/pod_with_special_fsGroup.json]: Please don't use a personal image.

File: pods/pod_with_special_supplementalGroups.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "name": "hello-pod",
+        "labels": {
+                "name": "hello-pod"
+        }
+  },
+
+  "spec": {
+      "containers": [{
+        "name": "hello-pod",
+        "image": "bmeng/hello-openshift",
```
Comment (wsun1) [inline pods/pod_with_special_supplementalGroups.json]: Same with above comments.

**Comment by zhouying7780** [comment]
add json for case 510612 and pod with special fsGroup and supplementalGroups json

**Comment by zhouying7780** [comment]
@wsun1 have updated to use public images, thanks.

**Pull request merged**